cms/djangoapps/contentstore/tests/test_libraries.py

```diff
@@ -847,3 +847,27 @@ def test_persistent_overrides(self):
         self.assertEqual(self.problem_in_course.display_name, new_display_name)
         self.assertEqual(self.problem_in_course.weight, new_weight)
         self.assertEqual(self.problem_in_course.data, new_data_value)
+
+
+class TestIncompatibleModuleStore(LibraryTestCase):
+    """
+    Tests for proper validation errors with an incompatible course modulestore.
+    """
+    def setUp(self):
+        super(TestIncompatibleModuleStore, self).setUp()
+        # Create a course in an incompatible modulestore.
+        with modulestore().default_store(ModuleStoreEnum.Type.mongo):
+            self.course = CourseFactory.create()
+
+        # Add a LibraryContent block to the course:
+        self.lc_block = self._add_library_content_block(self.course, self.lib_key)
+
+    def test_incompatible_modulestore(self):
```

Comment:
@andy-armstrong This is done in common/test/acceptance/tests/studio/test_studio_library_container.py . I would have put this test there, but it was not easy to swap out the modulestore once in a bok_choy test.